generators/entity-client/templates/angular/src/main/webapp/app/entities/_entity-popup.service.ts

```diff
@@ -24,6 +24,7 @@ import { DatePipe } from '@angular/common';
 <%_ } _%>
 import { <%= entityAngularName %> } from './<%= entityFileName %>.model';
 import { <%= entityAngularName %>Service } from './<%= entityFileName %>.service';
+import { HttpResponse } from '@angular/common/http';
```

Comment:
move this above local imports plz
We group imports like below, plz follow that everywhere

// libraray imports

// local imports